### DISCUSSION TITLE
Is there a way of dynamically changing the application name?

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2592] [Q&A]
What i mean is - you open the application and it changes it's name throughout the gameplay. Like, for example - showing the level number in the name

**Comment by RichardBray** [comment]
Hmmm, I can't say it's something I've tried. Do you know if there's a game that you've played that has done this before?

I think it's possible to do for a HTML5 game if you know some Javascript but I don't know if there's a way to do it via HaxeFlixel itself.

**Comment by Geokureli** [comment]
Assuming you want to do this on native desktop, if there was a way to do this, it wouldn't be through some HaxeFlixel utility. It would be done by something in [haxe's cpp package](https://api.haxe.org/cpp/index.html), if not in [openfl.Lib.application](https://api.haxeflixel.com/lime/app/Application.html). and if both of those fail I would see how to do it in cpp and inject some untyped cpp code into haxe.

You'll probably find better help with this in either [the haxe forum](https://community.haxe.org/) or [the openfl forum](https://community.openfl.org/)
